config_spec.toml

```diff
@@ -37,10 +37,15 @@ type = "String"
 optional = true
 doc = "The hex encoded macaroon to pass directly into LNDK"
 
+[[param]]
+name = "data_dir"
+type = "String"
+doc = "The path to the lndk data directory. By default this is stored in ~/.lndk"
+
 [[param]]
 name = "log_dir"
 type = "String"
-doc = "The path to the lndk log file. By default this is stored in ~/.lndk"
+doc = "The path to the lndk log file. If not specified, defaults to `data_dir/lndk.log`."
```

Comment:
small nit but maybe we could have data_dir in parentheses to indicate it's not literally data_dir. seems obvious but just in case :)

Reply:
Good idea -- done!